src/custom/utils/operator.ts

```diff
@@ -149,11 +149,17 @@ export async function getFeeQuote(chainId: ChainId, tokenAddress: string): Promi
   // TODO: Let see if we can incorporate the PRs from the Fee, where they cache stuff and keep it in sync using redux.
   // if that part is delayed or need more review, we can easily add the cache in this file (we check expiration and cache here)
 
-  const response = await _get(chainId, `/tokens/${tokenAddress}/fee`)
+  let response: Response | undefined
+  try {
+    const responseMaybeOk = await _get(chainId, `/tokens/${tokenAddress}/fee`)
+    response = responseMaybeOk.ok ? responseMaybeOk : undefined
+  } catch (error) {
+    // do nothing
+  }
 
-  if (!response.ok) {
+  if (!response) {
     throw new Error('Error getting the fee')
+  } else {
+    return response.json()
   }
```

Comment:
I'm not sure I like how this looks
What about
```ts
try {
    const response = await _get(chainId, `/tokens/${tokenAddress}/fee`)
    if (!response.ok) throw new Error('Error getting the fee')
    return response.json()
  } catch (error) {
     console.error('Error getting the fee', error)
     throw error
  }

Reply:
I agree here, I am a bit confused by the empty catch and if/then throw

Reply:
But the code is not equivalent

In yours I'm not logging errors in the case of `response.ok === false`, but since you throw and then catch, you log the error. Probably it will be logged also later (so 2 times)

Also yours doesn't address what I wanted to do here, that if I get an error in the request connection, the message is still custom (not the one displayed in the image). 

I did this just for not writing the same error twice. Empty catches in error handling can be justified, although requires a comment like I added. I think it's pretty easy to read, so I consider it a nitpick, and give in my last day I'll merge and you PR on top with whatever is u find convenient. There was also other alternative, to catch using the `.catch()` method of the promise. But I think we are splitting hairs here

Reply:
Merging this without the approvals, I don't want to leave it open this silly PR. Feel free to PR on top if u strongly disagree